resources/metadata/whitesourceExecuteScan.yaml

```diff
@@ -542,6 +542,17 @@ spec:
           - STEPS
         aliases:
           - name: npm/defaultNpmRegistry
+      - name: npmIncludeDevDependencies
```

Comment:
Do we want to set any default values for this new parameter https://github.com/SAP/jenkins-library/blob/b6b366066f1ff819bb2b8f43c18e0a748ba432a4/pkg/whitesource/configHelper.go#L151?

Reply:
Based on the testing this is the way where the parameter value gets assigned. 
If you don't have this param or you set it false - that would be default behavior 
If yes then param would be applied.